zipline/_protocol.pyx

```diff
@@ -76,6 +81,289 @@ cdef class BarData:
             self.data_frequency
         )
 
+    def spot_value(self, assets, fields):
+        """
+        Returns the spot value of the given assets for the given fields
+        at the current simulation time.  Spot values are the as-traded price
+        and are not adjusted for any events like splits or dividends.
+
+        Parameters
+        ----------
+        assets : Asset or iterable of Assets
+
+        fields : string or iterable of strings.  Valid values are: "price",
+            "last_traded", "open", "high", "low", "close", "volume"
+
+        Returns
+        -------
+        Float, pandas Series, or pandas DataFrame.  See notes
+        below.
+
+        Notes
+        -----
+        If a single asset and a single field are passed in, a scalar float
```

Comment:
This might be a case where putting a table in here like the one @ehebert commented with would be the most helpful way to describe this behavior.  (Sphinx can also auto-generate nice HTML renderings for tables embedded in docstrings.)